### ISSUE TITLE
Process killed

### ISSUE THREAD

**Issue by igorwhite** [madMAx43v3r/chia-plotter#75]
Hello!
Process killed Table 3 took

Xeon 2x2697v2
RAM 128Gb
RAM Disk 115Gb

./chia_plot -p ad1554d089591adb1c97d4ca7823492695b29ca9b90f159cae845a1 -f 93c5b71cdd703b2bee41bb3682c728aa41266ea554bba78ec5 -n 4 -r 46 -t /media/wzor/Data-1/Data1/ -2 /home/wzor/Data-2/ -d /media/wzor/Data-3/
Final Directory: /media/wzor/Data-3/
Number of Plots: 4
Number of Threads: 46
Number of Sort Buckets: 2^7 (128)
Pool Public Key:   ad1554d089591adb10320faebb4eaac97d4ca7823492695b29ca9b59cae845a1
Farmer Public Key: 93c5b71cdd703b2bee41bb34c002a87a302559171c728aa41266ea554bba78ec5
Working Directory:   /media/wzor/Data-1/Data1/
Working Directory 2: /home/wzor/Data-2/
Plot Name: plot-k32-2021-06-09-16-53-18077f65abffccba22ef02bb2d23849a1b49e225b4e9062490bb7164ab569f9b
[P1] Table 1 took 11.5127 sec
[P1] Table 2 took 226.067 sec, found 4294921038 matches
[P1] Table 3 took 316.4 sec, found 4294863936 matches
Killed

Where can I see why the process is being killed?

**Comment by madMAx43v3r** [comment]
Out of Memory? `dmesg | grep OOM`

**Comment by igorwhite** [comment]
Yes, not enough memory

**Comment by madMAx43v3r** [comment]
try less threads, jeeze, 46 will use quite some RAM, try 16 and go from there

**Comment by igorwhite** [comment]
Yes, reducing the number of threads helped. Thanks for a great job. I will definitely donate.

**Comment by Xefier** [comment]
Also increasing the number of buckets decreases the memory usage per thread but increases the disk I/O
